File: LENTABP10/src/main/java/com/lenta/lentabp10/MobileObjectsService/models/MB_S_36.java
package com.lenta.lentabp10.MobileObjectsService.models;

import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;

// Модель справочника настроек заданий на списание WOB (справочник текстов ГИС-контроля)
public class MB_S_36 {
    /// Тип контроля задания на списание
    @JsonProperty("TASK_CNTRL")
    @Getter private String taskCntrl;

    /// Текст длиной 40 знаков
    @JsonProperty("CNTRL_TXT")
    @Getter private String cntrlTxt;

    public MB_S_36(String taskCntrl, String cntrlTxt) {
        this.taskCntrl = taskCntrl;
        this.cntrlTxt = cntrlTxt;
    }
}

File: writeoff_breakage/src/main/java/com/lenta/bp10/mobileObjectsService/models/MB_S_36.kt
package com.lenta.bp10.mobileObjectsService.models

import com.google.gson.annotations.SerializedName

// Модель справочника настроек заданий на списание WOB (справочник текстов ГИС-контроля)
class MB_S_36(
        /// Тип контроля задания на списание
        @SerializedName("TASK_CNTRL")
        val taskCntrl: String,

        /// Текст длиной 40 знаков
        @SerializedName("CNTRL_TXT")
        val cntrlTxt: String
)